xmtp/components/Conversation/MessageComposer.tsx

```diff
@@ -2,19 +2,28 @@ import React, { useCallback, useEffect, useState } from 'react'
 import classNames from '../../helpers/classNames'
 import messageComposerStyles from '../../styles/MessageComposer.module.css'
 import { useRouter } from 'next/router'
+import { BezierSpinner } from '../../../components/Spinner/BezierSpinner'
+import { Web3Storage } from 'web3.storage'
 
 type MessageComposerProps = {
   onSend: (msg: string) => Promise<void>
 }
 
 const MessageComposer = ({ onSend }: MessageComposerProps): JSX.Element => {
+  const token = process.env.NEXT_PUBLIC_WEB3STORAGE
```

Comment:
This is were you upload the files right?
I would be super useful to save the `file.name` and `file.size` too, so later the recipient can see what he got before he clicks the link :)